vnext/Microsoft.ReactNative.Cxx/DesktopWindowBridge.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <ReactNotificationService.h>
+#include <ReactPropertyBag.h>
+#include <winrt/Windows.Foundation.h>
+#include <functional>
+
+#if defined(USE_WINUI3) && !defined(CORE_ABI) && !defined(__APPLE__) && \
+    WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM | WINAPI_PARTITION_GAMES)
+
+#include <winrt/Microsoft.ReactNative.h>
+
+namespace Microsoft::ReactNative {
+
+inline auto GetPropertyNameForWindowMessage(UINT uMsg) {
```

Comment:
>) [](start = 53, length = 1)

noexcept? Consider to add noexcept to other methods below if an exception is not really expected.